src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -8,10 +8,16 @@ redirects:
   - /docs/new-relic-only/tech-writer-style-guide/processes-procedures/formatting-terminal-commands
   - /docs/style-guide/writing-guidelines/formatting-terminal-commands/
   - /docs/style-guide/quick-reference/code-examples/
+  - /docs/code-formatting-guidelines
+  - /docs/new-relic-only/basic-style-guide/writing-guidelines/code-formatting-guidelines
+  - /docs/style-guide/writing-guidelines/code-formatting-guidelines-var-mark
+  - /docs/style-guide/formatting/code-formatting-guidelines-var-mark
 ---
 
 We use a variety of formatting to highlight code or other technical language. You can use the `<code>`, `<pre>`, and `<var>` tags to indicate "raw" technical content such as excerpts from a config file, an API method name, or a file path.
```

Comment:
@austin-schaefer, I think you could remove this second sentence of this paragraph since we're no longer using `<code>` and `<pre>`. Plus the following table has examples, right?

Reply:
Good call!